Source/Core/DolphinWX/ISOProperties/FilesystemPanel.cpp

```diff
@@ -68,20 +68,39 @@ wxImageList* LoadIconBitmaps(const wxWindow* context)
   return icon_list;
 }
 
+// Iterates over all files in a directory summing up their sizes to determine the size of the directory
+u64 DetermineDirectorySize(const DiscIO::FileInfo& directory, u64 size)
+{
+  for(const DiscIO::FileInfo& file_info : directory)
+  {
+    if(file_info.IsDirectory())
+    {
+      size += DetermineDirectorySize(file_info,0);
+    }
+    else
+    {
+      size += file_info.GetSize();
+    }
```

Comment:
Since both the `if` and `else` paths only contain one statement each, you should write them without the curly braces, like so:

```cpp
for (const DiscIO::FileInfo& file_info : directory)
{
  if (file_info.IsDirectory)
    size += DetermineDirectorySize(file_info, 0);
  else
    size += file_info.GetSize();
}
```

There are also a few lint violations. I'll start up the lint bot for you so that it'll list them all.